packages/generator/tsconfig.json

```diff
@@ -10,6 +10,9 @@
     ".eslintrc.cjs"
   ],
   "references": [
+    {
```

Comment:
I think we wouldn't need these if we added `types` to the package.json (I think conditional exports don't work super well and might be why you needed to add these)